pkg/sdn/plugin/vnids_node.go

```diff
@@ -26,10 +24,7 @@ type nodeVNIDMap struct {
 }
 
 func newNodeVNIDMap() *nodeVNIDMap {
-	return &nodeVNIDMap{
-		ids:        make(map[string]uint32),
-		namespaces: make(map[uint32]sets.String),
-	}
+	return &nodeVNIDMap{}
```

Comment:
No, This enables nil check (https://github.com/openshift/origin/pull/11137/files#diff-eabef4ae7ed24f933a3d3ac531af3814R65)
and GetVNID (https://github.com/openshift/origin/pull/11137/files#diff-5f233cf7267651424a495cc6b900f8d6R230) will return correct value both for subnet and multitenant plugins.